siac/hostcmd.go

```diff
@@ -55,8 +56,7 @@ func hostconfigcmd(param, value string) {
 		// convert price to hastings/byte/block
 		p, ok := new(big.Rat).SetString(value)
 		if !ok {
-			fmt.Println("could not parse price")
-			return
+			dieNoError("Could not parse price")
```

Comment:
maybe something like:
```go
// die writes its arguments to os.Stderr (separated by newlines), and
// then exits with the default error code.
func die(args ...interface{}) {
    for _, arg := range args {
        fmt.Fprintln(os.Stderr, arg)
    }
    os.Exit(exitCodeGeneral)
}
```
It is a bit unconventional though. In general I think it's best to leave formatting up to the caller, but this seems like a special case.